driver-core/src/main/java/com/datastax/driver/core/TypeCodec.java

```diff
@@ -827,6 +831,222 @@ public Date deserialize(ByteBuffer bytes) {
         }
     }
 
+    static class SimpleDateCodec extends TypeCodec<Integer> {
```

Comment:
I'd rather expose this CQL type as a java.util.Date. Returning the raw number of days since the epoch is not very user-friendly.